truffle/src/com.oracle.truffle.runtime/src/com/oracle/truffle/runtime/OptimizedRuntimeOptions.java

```diff
@@ -158,6 +158,14 @@ public ExceptionAction apply(String s) {
     // TODO: GR-29949
     public static final OptionKey<Long> CompilerIdleDelay = new OptionKey<>(10000L);
 
+    @Option(help = "Before the Truffle runtime submits an OptimizedCallTarget for compilation, it checks for the " +
+                   "compilation activity mode in the host VM. If the activity mode is 'STOP_COMPILATION' because " +
+                   "of a full code cache, no new compilation requests are submitted and the compilation queue is flushed. " +
+                   "After 'StoppedCompilationRetryDelay' milliseconds new compilations will be submitted again " +
+                   "(which might trigger a sweep of the code cache and a reset of the compilation activity mode in the host JVM).",
```

Comment:
Well this sounds like it would be supported in all runtimes. But it is not. SVM does not support this yet. So we should mention this only works for HotSpot right now. Note that OptimizedRuntimeOptions are shared across all optimizing runtimes this includes SVM.